packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {ACTIVE, RECONNECTION} from '../constants';
+import ReconnectionError from '../errors/reconnection';
+import PeerConnectionManager from '../peer-connection-manager';
+
+const ReconnectionManager = {};
+
+ReconnectionManager.status = RECONNECTION.STATE.DEFAULT_STATUS;
```

Comment:
please add opinions on the 3 state managed values in a singleton -- was thinking maybe creating an object instead, but singleton makes more sense for reconnection since it can only be used by one call at one time, but maybe need to create a state machine to manage it's values rather than store them directly on the singleton

Reply:
awesome works for me.